### PR TITLE
[CI] Disable flash_attn backend for spec decode

### PR DESCRIPTION
**Pull request by simon-mo**
See #5152

### CODE REVIEW

**Review by comaniac — APPROVED** [review]
Thanks for unblocking CI. Out of curiosity, do we know the root cause of the failure?

**Comment by simon-mo** [comment]
Nope. Please help debug if you have bandwidth!

**Review by pcmoritz — APPROVED** [review]
Thanks for unblocking the CI!

File: .buildkite/test-pipeline.yaml [hunk 1]
```diff
@@ -45,7 +45,7 @@ steps:
   - TEST_DIST_MODEL=meta-llama/Llama-2-7b-hf DISTRIBUTED_EXECUTOR_BACKEND=mp pytest -v -s distributed/test_basic_distributed_correctness.py
   - TEST_DIST_MODEL=facebook/opt-125m DISTRIBUTED_EXECUTOR_BACKEND=mp pytest -v -s distributed/test_chunked_prefill_distributed.py
   - TEST_DIST_MODEL=meta-llama/Llama-2-7b-hf DISTRIBUTED_EXECUTOR_BACKEND=mp pytest -v -s distributed/test_chunked_prefill_distributed.py
-  - pytest -v -s spec_decode/e2e/test_integration_dist.py 
+  - pytest -v -s spec_decode/e2e/test_integration_dist.py
```
Comment (cadedaniel) [inline .buildkite/test-pipeline.yaml:48]: should we do the same for the tp>1 tests?

**Review by cadedaniel — APPROVED** [review]
Thanks for fixing!

File: .buildkite/test-pipeline.yaml (hunk 1 above)
Comment (simon-mo) [reply .buildkite/test-pipeline.yaml:48]: It did pass without the flag 🤔 so I'll leave it be

**Comment by WoosukKwon** [comment]
🤦

**Pull request merged**